api/cloud_accounts.go

```diff
@@ -117,6 +117,13 @@ func (svc *CloudAccountsService) List() (response CloudAccountsResponse, err err
 	return
 }
 
+// ListByType lists the cloud accounts from the provided type that are available
```

Comment:
Do we need an api test for this in the `api_test` package?  There is a `cloud_accounts_test.go` established in the api folder...